### ISSUE TITLE
Focus outline of active tags is not controlled by FocusStyleManager

### ISSUE THREAD

**Issue by invliD** [palantir/blueprint#2575]
#### Environment

- __Package version(s)__: 1.x

#### Steps to reproduce

1. Render a `<Tag className="pt-active" />`
2. Call `FocusStyleManager.onlyShowFocusOnTabs();`

#### Actual behavior

Tag has a focus outline

#### Expected behavior

Tag doesn't have a focus outline

#### Possible solution

Add an override for the `.pt-tag.pt-active` focus outline to the `.pt-focus-disabled` class.

**Comment by llorca** [comment]
Fixed in #2428

**Comment by invliD** [comment]
Not in 1.x, which is the version I filed this issue for.

**Comment by llorca** [comment]
Doesn't matter, it's fixed in 2.x and 3.x. If you'd like to leave this open, rename the issue and mention back porting

**Comment by invliD** [comment]
I mentioned 1.x in the PR description. Backporting is a part of the solution, not the issue. This is an issue.

**Comment by giladgray** [comment]
sorry, not backporting this now that 3.0 is out.
